framework/doc/content/framework/reviewing.md

```diff
@@ -0,0 +1,34 @@
+# Code Review
+
+## Review Guidelines
+
+- Contributors should be regarded as innocent until proven guilty
+  - Contributors probably aren't out to destroy MOOSE
+  - The contributor has probably thought about their contribution a lot
+- Reviews should be thoughtful
+- Reviews should be thorough; we don't want bugs! But
+  - Don't let perfect be the enemy of the good
+  - We'd like development to be agile
+  - Merged code will have eyes of tens of MOOSE contributors; runs of hundreds
+    to thousands of users
+  - Unmerged code will have eyes of much fewer contributors and runs of much
+    fewer users
+    - No code, no bug reports! But also no features
+  - People will be hesitant to contribute if the review process is onerous
+- Are there things we can do to abate fear of bugs?
+  - Request more tests
+- As a reviewer, try to make it clear to the author what conversations are
+  required changes vs. suggestions (perhaps GitHub Requested Change vs Comment?)
+- NQA-1 explicitly allows for a change-control board member to push commits to a
+  pull request and maintain independent reviewer status as long as the pushed
+  commits are not "significant."
+
+## Addressing Deadlock Between a CCB Author and CCB Reviewer
+
```

Comment:
maybe this should be a subsection and not its own big category. This doesnt happen often, and would happen even less with a tad more politeness